client/my-sites/stats/activity-log-banner/error-banner.jsx

```diff
@@ -76,8 +76,10 @@ class ErrorBanner extends PureComponent {
 					details: translate( 'We came across a problem while trying to restore your site.' ),
 				}
 			: {
-					title: translate( 'Problem creating a backup' ),
-					details: translate( 'We came across a problem creating a backup for your site.' ),
+					title: translate( 'Problem creating a file' ),
+					details: translate(
+						'There was a problem building your backup into a downloadable file.'
```

Comment:
for what it's worth I think we have used "prepare" in many places, or at least have behind the scenes

```
There was a problem preparing your backup for downloading. blah blarg blurz
```